nltk/collocations.py

```diff
@@ -395,12 +395,12 @@ def demo(scorer=None, compare_scorer=None):
     from nltk.metrics import BigramAssocMeasures
 
     try:
-        scorer = eval("BigramAssocMeasures." + sys.argv[1])
-    except IndexError:
+        scorer = getattr(BigramAssocMeasures, sys.argv[1])
+    except (IndexError, AttributeError):
         scorer = None
     try:
-        compare_scorer = eval("BigramAssocMeasures." + sys.argv[2])
-    except IndexError:
+        compare_scorer = getattr(BigramAssocMeasures, sys.argv[2])
+    except (IndexError, AttributeError):
         compare_scorer = None
 
     demo(scorer, compare_scorer)
```

Comment:
To make the CLI behavior clearer when both scorer names are missing/invalid, you could handle that case explicitly and show a usage message:

```suggestion

    if scorer is None and compare_scorer is None:
        print("Usage: python -m nltk.collocations <scorer_name> [<compare_scorer_name>]")
        available = [n for n in dir(BigramAssocMeasures) if not n.startswith("_")]
        print("Available scorer names:", ", ".join(available))
        sys.exit(2)

    demo(scorer, compare_scorer)

Reply:
This would change the default behaviour. Currently when we run
```bash
python -m nltk.collocations
```
it runs the demo with with defaults: scorer=likelihood_ratio, compare_scorer=raw_freq

Prefer to avoid this change within the scope of this PR.